module/clock/src/mod_clock.c

```diff
@@ -278,10 +280,10 @@ static int clock_process_pd_pre_transition_notification(
     };
 
     pd_params = (struct mod_pd_power_state_pre_transition_notification_params *)
-        event->params;
+        (void *)event->params;
```

Comment:
Which compiler are you using? Could you please let us know the exact compilation warning here, so we can put this in our internal checks if missing.

Reply:
I use a cortex-A toolchain, since building for cortex-A OP-TEE. Currently GCC 8.3:
`arm-linux-gnueabihf-gcc (GNU Toolchain for the A-profile Architecture 8.3-2019.03 (arm-rel-8.36)) 8.3.0`

Reply:
Could you paste the error? I'm unable to reproduce this with ARM64 GCC 8.2: https://gcc.godbolt.org/z/B-A-F5

Reply:
well, I checked back and... did not produce any build warning (nor issue).
I'll discard these changes.